_clients/python-low-level.md

```diff
@@ -98,6 +98,41 @@ client = OpenSearch(
 ```
 {% include copy.html %}
 
+If you need to verify ssl to a different hostname, specify it in `ssl_assert_hostname`: 
+
+```python
+host = 'localhost'
+port = 9200
+
+# Create the client with SSL/TLS and hostname verification disabled.
+client = OpenSearch(
+    hosts = [{'host': host, 'port': port}],
+    http_compress = True, # enables gzip compression for request bodies
+    use_ssl = True,
+    verify_certs = False,
```

Comment:
To enable hostname verification, **set verify_certs to True**. 

Please review this [PR](https://github.com/opensearch-project/opensearch-py/pull/694) at your convenience. You may also wait until it's merged to avoid potential revisions and include some content here.